pkgs/tools/typesetting/tex/tex-match/default.nix

```
@@ -0,0 +1,28 @@
+{ rustPlatform, fetchFromGitHub, gtk3, pkg-config, glib, lib }:
+
+rustPlatform.buildRustPackage {
+  pname = "tex-match";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "zoeyfyi";
+    repo = "TeX-Match";
+    rev = "65498b4013f493852276f3f918eeed35479f72d3";
```

This doesn't match the version attribute you specified above. Please either change this to

```suggestion
    rev = "v${version}";
```

or change the version attribute to "unstable-" + the date of this commit:
```nix
  version = "unstable-2021-01-12";
```